samples/net/stats/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _net_stats-sample:
+
+Network Statistics Sample Application
+#####################################
+
+Overview
+********
+
+This sample application shows how to query network statistics from user
+application.
+
+The source code for this sample application can be found at:
+:file:`samples/net/stats`.
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
+Building and Running
+********************
+
+A good way to run this sample application is with QEMU as described in
+:ref:`networking_with_qemu`.
+
+Follow these steps to build the network statistics sample application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/stats
+   :board: <board to use>
```

Comment:
Since only qemu_* boards are meaningful here, it would be good to mention that:

        :board: <qemu_board to use>

Reply:
Actually we are not restricted to only qemu with this sample. There is the sentence above this that says "A good way to run this sample application is with QEMU..." but we could run this with many different boards.

Reply:
True, but since this whole example talks about using QEMU, wouldn't it be good to use one of the qemu_* boards here?